src/store/modules/features.store.js

```diff
@@ -1,8 +1,15 @@
+import { containsCoordinate } from 'ol/extent'
+
 import EditableFeature, { EditableFeatureTypes } from '@/api/features/EditableFeature.class'
-import LayerFeature from '@/api/features/LayerFeature.class.js'
-import getProfile from '@/api/profile/profile.api.js'
+import { identify, identifyOnGeomAdminLayer } from '@/api/features/features.api'
+import LayerFeature from '@/api/features/LayerFeature.class'
+import getProfile from '@/api/profile/profile.api'
+import { flattenExtent } from '@/utils/coordinates/coordinateUtils'
 import { allStylingColors, allStylingSizes } from '@/utils/featureStyleUtils'
-import log from '@/utils/logging.js'
+import log from '@/utils/logging'
+
+const DEFAULT_FEATURE_COUNT_SINGLE_POINT = 10
+const DEFAULT_FEATURE_COUNT_RECTANGLE_SELECTION = 50
```

Comment:
Just wondering if a max of 25 would make more sense and be more performant ? Did you test locally with the prod backend to see if it is faster ? Currently using 50 is quite slow in comparison to the old map geoadmin.

Reply:
about 100 to 200ms to get 50 features from the PROD backend, then loading each features' detail takes about 500ms (so in less than 1sec, you get your 50 features rendered on the map).
That's "good enough" but the fact that we need to request each feature's detail adds quite a lot of time to the processing (we could maybe parallelize that in the code)